Files/Extensions/DateTimeExtensions.cs

```diff
@@ -10,7 +10,7 @@ public static string GetFriendlyDateFromFormat(this DateTimeOffset d, string ret
         {
             var elapsed = DateTimeOffset.Now - d;
 
-            if (d.Year == 1601)
+            if (d.Year == 1601 || d.Year == 9999)
```

Comment:
Should we manually display the years? I can't think of a useful use case, but it might be preferable to show the year instead of an empty space.

Reply:
> Should we manually display the years? I can't think of a useful use case, but it might be preferable to show the year instead of an empty space.

@yaichenbaum I've followed the 1601 case (#5269) since Windows doesn't show a creation / modification date on files or folders that doesn't have a creation date:

Files before my PR that changes this on 1601 case:
![image](https://user-images.githubusercontent.com/11770760/122640977-abf48280-d102-11eb-8e67-85e24ddae5ac.png)

Windows explorer on the same case:
![image](https://user-images.githubusercontent.com/11770760/122641159-b9f6d300-d103-11eb-9a3c-a0fdafa40eed.png)

I personally prefer an empty space than an unreal date as 1601 or 9999.